sorc/fsync_file.cd/fsync_file.c

```diff
@@ -1,13 +1,9 @@
-/** 
- * @file
- * Call fsync() on a file.
- */
 #include <stdio.h>
+#include <stdlib.h>
 #include <errno.h>
 #include <string.h>
 #include <unistd.h>
 #include <fcntl.h>
-#include <stdlib.h>
```

Comment:
OK, so there are no changes here except moving the include statement. I will move it...